lib/express_validator.js

```diff
@@ -332,7 +339,7 @@ function makeValidator(methodName, container) {
     }
 
     var args = [];
-    args.push(this.value);
+    args.push(this.value += '');
```

Comment:
It may make more sense to perform the conversion and assignment of `this.value` on [line 23](https://github.com/ctavan/express-validator/pull/229/files#diff-6c47d849cc78d146191d4d361699b452L23) so it need only be converted once at the source.

If that is not an option, I would suggest moving the assignment onto its own line in order to make it more explicit.
e.g.
``` javascript
this.value += '';
args.push(this.value);
```

On a more important note, this means of converting the value to a string is insufficient and introduces breaking changes. The behavior changes for values such as undefined, null, and NaN, and possibly for arrays and objects as others have noted. For instance, validator.js would coerce undefined, null, and NaN values to an empty string. Whereas this code coerces undefined to the string `'undefined'`.

As I mentioned above, I think it's important to preserve the same conversion rules for this PR so as to not introduce major changes. Therefore, I think @Jakeii's suggestion to mirror the code previously used by validator.js makes sense.